public/index.html

```diff
@@ -74,6 +74,52 @@
 
     </uxrow>
 
+    <hr/>
+
+
+    <uxrow>
+        <uxcol class="large-12">
+            <h2>Pricing Table demo</h2>
+        </uxcol>
+    </uxrow>
+    <uxrow>
+        <uxcol class="large-6">
+
+            <uxpricingtable title="Samsung Galaxy S6" price="$699.99" href="/buy/samsunggalaxys6/now" status="comingsoon">
```

Comment:
Interesting again - do we deliberately limit the title to text? I'll go with your suggestion for now